libs/brig-types/src/Brig/Types/Swagger.hs

```diff
@@ -375,11 +376,23 @@ invitationRequest = defineModel "InvitationRequest" $ do
         description "Locale to use for the invitation."
         optional
 
+role :: DataType
+role = Model.Prim $ Model.Primitive
+    { Model.primType     = Model.PrimString
+    , Model.defaultValue = Just defaultRole
+    , Model.enum         = Just [minBound..]
+    , Model.minVal       = Just minBound
+    , Model.maxVal       = Just maxBound
+    }
+
 invitation :: Model
 invitation = defineModel "Invitation" $ do
     description "An invitation to join Wire"
     property "inviter" bytes' $
         description "User ID of the inviter"
+    property "role" role $ do
+        description "User ID of the inviter"
```

Comment:
copy paste error.

Reply:
sorry.

...  third try?  (or is it 4th?)